packages/replayio/src/utils/recordings/getRecordings.ts

```diff
@@ -265,7 +264,7 @@ export function getRecordings(processGroupIdFilter?: string): LocalRecording[] {
     }
   }
 
-  debug("Found %s recordings:\n%o", recordings.length, recordings);
+  debug("Found %s recordings:\n%o\n%o", recordings.length, JSON.stringify(recordings, null, 2));
```

Comment:
You have too many formatters here (3 formatters but only 2 values) and `%o` is used while you stringify that object urself. You could use multi-line object formatter like this:
```suggestion
  debug("Found %s recordings:\n%O", recordings.length, recordings);
```